### PR TITLE
Feature/cicd deploy

### PR DESCRIPTION
**Pull request by jonathansmolenski**
This includes the github action code for CI/CD based on branch pushes.  The code deploys but currently there is no smoke test to determine that it deployed properly.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -67,25 +75,85 @@ jobs:
...
+      #     TF_VAR_sds_username:     ${{ secrets.SDS_USERNAME }}
+      #     TF_VAR_sds_password:     ${{ secrets.SDS_PASSWORD }}
+      #     TF_VAR_sds_rs_bucket:    ${{ vars.SDS_RS_BUCKET }}
+
+      #     TF_VAR_publish_bucket:   ${{ vars.PUBLISH_BUCKET }}
+      #   run: |
+      #     echo "${{ secrets.JPLICA_CERT }}" >> ${{ runner.temp }}/JPLICA.pem
+ 
+      #     source bin/config.sh sit
+      #     terraform apply -auto-approve
```
Comment (joshgarde) [inline .github/workflows/build.yml:120]: Just remove instead of commenting out

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -101,6 +169,6 @@ jobs:
           TF_VAR_publish_bucket:   ${{ vars.PUBLISH_BUCKET }}
         run: |
           echo "${{ secrets.JPLICA_CERT }}" >> ${{ runner.temp }}/JPLICA.pem
- 
-          source bin/config.sh sit
-          terraform apply -auto-approve
+
+          source bin/config.sh ${{ vars.TF_VENUE }}
```
Comment (joshgarde) [inline .github/workflows/build.yml:173]: Doesn't this need to be lowercased like here:
https://github.com/podaac/swodlr-user-notify/blob/682e19d2c8aaee8143fa8d24727698b00b32d33b/.github/workflows/build.yml#L169

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -101,6 +169,6 @@ jobs:
...
+          terraform apply -auto-approve
```
Comment (joshgarde) [inline .github/workflows/build.yml:174]: Missing new line

**Review by joshgarde — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml (hunk 2 above)
Comment (jonathansmolenski) [reply .github/workflows/build.yml:173]: It seems to not need that for some reason, but I think it's a good idea to apply the lowercase to everything

File: .github/workflows/build.yml (hunk 1 above)
Comment (jonathansmolenski) [reply .github/workflows/build.yml]: removed

**Review by jonathansmolenski — COMMENTED** [review]
Fixes have been applied

**Pull request merged**